netty/src/main/java/io/grpc/netty/NettyClientTransport.java

```diff
@@ -105,6 +112,8 @@ class NettyClientTransport implements ConnectionClientTransport {
   private final ChannelLogger channelLogger;
   private final boolean useGetForSafeMethods;
   private final Ticker ticker;
+  private final ConcurrentHashMap<String, Boolean> authoritiesAllowedForPeer =
```

Comment:
We'll want to limit the maximum size of this map. That may mean we swap it to a LinkedHashMap and protect it with a lock, so we can drop LRU.

Reply:
Should the max size be hardcoded to 10000? We don't need it to be configurable via channel params, right?

Reply:
I think hard-coding it to 100 would be fine. 10 is probably sufficient, so 100 should be plenty. If it is exceeded, it is almost certainly an unbounded number of authorities.

Reply:
Done.